apps/authentications/src/app/controllers/authentications.controller.v2.ts

```diff
@@ -48,4 +50,20 @@ export class AuthenticationControllerV2 {
       password
     );
   }
+
+  @ApiOkResponse({
+    type: CheckingResponse,
+  })
+  @Post('exists/castcle-id')
+  @HttpCode(200)
+  async checkCastcleIdExists(@Body() body: CheckIdExistDto) {
+    const user = await this.authenticationService.getExistedUserFromCastcleId(
+      body.castcleId
+    );
+    return {
+      payload: {
+        exist: user ? true : false, // true=มีในระบบ, false=ไม่มีในระบบ
```

Comment:
```suggestion
        exist: user ? true : false,
```
คอมเมนต์ไม่ต้องใส่มาครับ

Reply:
already fixed